index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+    <title>My Portfolio</title>
+</head>
+<body>
+    
+    <!-- navbar section -->
+<nav class="navbar">
+    <h3 id="logo">Dela McRaph</h3>
+    <ul>
+        <li><a href="#">Portfolio</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+    </ul>
+
+</nav>
+
+<!-- self introduction -->
+<div class="container">

```

Comment:
- [ ] Please always wrap your main content of the website with a `main` tag which again helps to improve the **accessibility** of your website and it will help to cover a wide `audience` all around the world. As I mentioned it the part of the `requirement` to follow best practices so please fix it.